EvolutionModel/Foodbag.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Evolution.Model {
+    public class Foodbag {
+        public uint Foodamount {
+            get;
+            private set;
+        }
+
+        public Foodbag(){ 
+            Foodamount = 0;
+        }
+
+        public uint Count => Foodamount;
```

Comment:
Deze is niet meer nodig. Gebruikers (alles wat buiten de class zit) kan direct Foodamount aanroepen. Count kan dus weg, want dubbelop.